### ISSUE TITLE
Split out chain/node querying from /status into separate routes

### ISSUE THREAD

**Issue by jnaviask** [hicommonwealth/commonwealth#4933]
## Description

Follow-up to discussion on #4910. Currently we load data for all communities upfront in the /status route. This approach becomes unsustainable as the site grows, because (a) we have no need to load the data for all communities upfront, when we are only ever in one community at a time, and (b) because /status must load for all pages on the application, so growth will massively slow down the loading experience.

## Engineering Requirements

The proposed solution is to split out the "chainsWithSnapshots" and "nodes" objects from the /status route into separate routes, which to begin with will simply fetch the same data without options/flags. Within scope of this ticket, we should query these routes in parallel with /status, to avoid requiring major front-end changes (i.e. we get the same data, but make a few queries rather than just one /status query).

Later, but out of scope for this ticket, we can extend the functionality of these routes + how they interact with the client (e.g. using react-query for `app.config.nodes` and `app.config.chains`, lazy loading requests for chain metadata, etc).

**Comment by rbennettcw** [comment]
Estimating as a 2 to refactor + add tests

**Comment by CowMuon** [comment]
<3 this estimate, thanks Ryan. 

@gdjohnson What of visibility is needed in documentation here? In particular would like to surface impact of chainsWithSnapshots and especially for new devs on-boarding onto the project, who may not know what snapshots are (or how nodes are handled, or even our development trajectory away from _loading-all-the-things_ to a more strategic approach of loading only what's needed (or has high probability of being needed for the next thing the user is likely to do.) 

The headline here is that as we move from "architecture in the rear view" (ie architectural modeling based on accepted commits) to (well docuented) architectural _planning_, we need to insure Documentation is highly impactful to the engineers it is aimed at, and not merely chasing the tail of what's been done (which would just reassign the 'rearview' function from development to documentation.) @gdjohnson
